src/routes/index.ts

```diff
@@ -42,4 +129,47 @@ router.get('/contacts', async (req, res) => {
     console.log('got contacts');
     res.end();
 });
+
+router.post('/contacts', async (req, res) => {
+
+    const contact = await getContacts();
+    res.json(contact);
+    console.log('got contacts');
+    res.end();
+});
+
+
+//returns a list of Saasquatch participants for a given api key and tenet alias
+const getParticipants = async () => {
+    try {
+        console.log('=== attempting to get all SaaSquatch participants api key is ===' + SAPIKEY);
+        const allParticipants = 'https://staging.referralsaasquatch.com/api/v1/' +STENANTALIAS+ '/users';
+        console.log(allParticipants);
+        const response = await axios.get(allParticipants, {
+            auth: {
+                username: '',
+                password: SAPIKEY
+            }
+        });
+        console.log('test' + response);
+        const data = response.data;
+        console.log(data);
+        return data;
+    } catch (e) {
+        console.error('  > Unable to retrieve participants');
+        console.log(e);
+        return e;
+    }
+	
+
+}
+
+router.get('/participants', async (req, res) => {
+
+    const Participants = await getParticipants();
```

Comment:
Perhaps a try-catch here? 
example: (which can be a scenario after we are using access tokens for API calls)  user uninstalls app, access token is not valid and refresh token is also invalid so it can't be used to make authenticated API calls. We must redirect user to authorize the application again. So in the catch we can have res.redirect("/hubspot")